docs/en/loading/Flink-connector-starrocks.md

```diff
@@ -379,6 +379,10 @@ DISTRIBUTED BY HASH(`id`);
     ./bin/start-cluster.sh
     ```
 
+#### Network configuration
+
+Ensure that the machine where Flink is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/cb49c819-6307-465b-ac71-afaa93c0730a)
It's better to remove "ports".